### PR TITLE
Added Part 3.1 on Anaconda "Case"

### PR DESCRIPTION
**Pull request by leriomaggio**
Ended up adding 4 slides in total: 

- intro Anaconda as a company and emphasis on RemoteFirst and Distributed Teams
- CR in Anaconda briefly (how's important) 
- CR in Anaconda: Authors' view
- CR in Anaconda: Reviewers' view

please let me know what you think / would like to retain.

### CODE REVIEW

File: slides/slides.md [hunk 1]
```diff
@@ -550,3 +550,77 @@ guard against these.
...
+
+- The type of the team is very important, for example:
+	+ All IT-experts; IT people w/ diff experience or backgrounds; IT & non-IT people.. 
+
+### Code Review in Anaconda
+
+(Author's Checklist)
+
+- **Always** communicate up-front changes to an existing code base, i.e. _Open an Issue in the repo_
+
```
Comment (tlestang) [inline slides/slides.md:599]: Ok this point is very interesting and will give weight to some "good practices" introduced in the previous section.

File: slides/slides.md [hunk 2]
```diff
@@ -550,3 +550,77 @@ guard against these.
...
+	+ Provide a welcoming and friendly  _onboarding_ for newcomers 	
+	
+. . .
+
+- Code Review sets and (easily) adapts differently depending on specific Projects requirements
+
+. . . 
+
+- The type of the team is very important, for example:
+	+ All IT-experts; IT people w/ diff experience or backgrounds; IT & non-IT people..
```
Comment (tlestang) [inline slides/slides.md:592]: I guess what is particularly interesting in our case is HOW doe Anaconda (as an organisation) achieves (enforce?) such levels of adaptation.

**Comment by tlestang** [comment]
Thanks @leriomaggio Unfortunately given the short timeline I don't have much time to look at this in much details. I do think this is valuable, especially is you keep two things in mind:
1. Our presentation is focusing on research software (it's in the title after all), so it would be good to connect your part to  this context. Perhaps some of the practices you mention are applicable.
2. We're already close to the time limit and adding 4 slides makes me a bit nervous. I'll do my best to shorten the two previous parts in order to make room for yours, but you'll probably have a between 2 and 5 minutes minutes overall.

**Review by dokato — COMMENTED** [review]
Just to piggyback on @tlestang points. I really like the content of the slides, so all for keeping it and merging. Anaconda works on the verge of pure industrial applications and research so that will give a unique perspective.

Having said that, I agree that with 4 slides you would need to rush or shorten it a little bit. I am sure that there'll be an opportunity to discuss more during the Q&A.

File: slides/slides.md [hunk 3]
```diff
@@ -550,3 +550,77 @@ guard against these.
...
+. . .
+
+- Code Review sets and (easily) adapts differently depending on specific Projects requirements
+
+. . . 
+
+- The type of the team is very important, for example:
+	+ All IT-experts; IT people w/ diff experience or backgrounds; IT & non-IT people.. 
+
+### Code Review in Anaconda
```
[the remark this answers is not in the document]
Comment (leriomaggio) [reply slides/slides.md:594]: Yes that's correct! thank you :)

**Comment by leriomaggio** [comment]
Thank you both for all the comments. I am not sure what we decided to do with those slides then. I guess they've been left out - shall we keep those for the Q&A then? :) 
We could even use them as an icebreaker or to fill in if no questions are coming. 
We could get rid of the "company" slide and keep only the one about remote/async and CR example?

**Comment by dokato** [comment]
Sorry @leriomaggio for late reply. Busy day.

>  We could get rid of the "company" slide and keep only the one about remote/async and CR example?

I think given the time contstraints it's a fair approach. Do you want to do that? I am happy to merge it in then, and pass on to you during talk to add a few cents on that. Does it sound good?

**Comment by leriomaggio** [comment]
No worries at all - that sounds like a great plan! I am going to do the fixings to the slides, will send an updated PR so that we can merge.

**Comment by tlestang** [comment]
> We could get rid of the "company" slide and keep only the one about remote/async and CR example?

That works for me. @leriomaggio 

I could also bring you in at 2 moments in the second part of the talk:
- After discussing good practices for authors, you could tell about how authors at Anaconda are expected to communicate their changes upfront, provide a good description etc... (based on you 3rd slide)
- After I mention the importance of a policy and a checklist. I could litteraly just mention it and give you the floor for a couple of minutes. (based on your 4th slide)

**Comment by leriomaggio** [comment]
Hey @tlestang thanks for your comment. 
I guess that it would make sense in terms of content, but it would feel a bit weird that I jump into the discussion twice, just to comment one slide and then pass back to you - it will also disrupt the flow. 

My suggestion is that we keep those three slides in the end, self-contained and presented as conclusive remarks/use cases from industry. 
I will mention the few words about the company in the title slide (i.e. Part 3.1), which, btw, will just rename "Code Review in OSS" and then will talk about it. 

Hopefully, all the best practice mentioned won't be "far away" enough in the discussions so that it won't be difficult to follow and make links. 

Pls, let me know what you think. 
Valerio

**Comment by leriomaggio** [comment]
@tlestang @dokato all latest changes pushed. If you could just add the Anaconda logo to the title slide for last part, that would be very much appreciated! 

As a side note: I've noticed that part two has a dark grey bg colour with black text - it's barely readable. 
Telling you in case you didn't notice. 

Thanks

**Comment by tlestang** [comment]
> My suggestion is that we keep those three slides in the end, self-contained and presented as conclusive remarks/use cases from industry.

Works for me.Then ignore #16

**Comment by tlestang** [comment]
For the Anaconda logo please create a 1080 x 1920 image with transparent background and Anaconda logo in a corner. We can then make it the slide background

**Comment by dokato** [comment]
Haha, "too many cooks spoil the broth "...

@leriomaggio I also integrated your slides to avoid this conflict here in my latest PR #17 . Check whatever suits you and give a thumbs up to merge if you can't do it yourself. I agree that your part at the end will be better for the flow. You can manage the discussion in Q&A too then.

PS I fixed the background on main, to speed things up.

**Comment by leriomaggio** [comment]
@dokato checked and reviewed #17 - LGTM :) left just a comment on background image, if you can. No worries if it's too complicated to fix quickly :)

**Comment by leriomaggio** [comment]
Closed this as everything that it needed will be integrated in #17 ;)

**Pull request closed without merging**